app/src/main/res/layout/view_slide_list_row.xml

```diff
@@ -9,8 +9,7 @@
         android:orientation="horizontal"
         android:padding="8dp">
 
-        <ImageView
-            android:id="@+id/slide_list_row_image"
+        <ImageView android:id="@+id/slide_list_row_image"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/SlideViewer/blob/50f2df2e17a27ff307775eeb1f1eb307b9a8c17e/app/src/main/res/layout/view_slide_list_row.xml#L12"></span>[ContentDescription] [Accessibility] Missing <code>contentDescription</code> attribute on image<br />
 Non-textual widgets like ImageViews and ImageButtons should use the <code>contentDescription</code> attribute to specify a textual description of the widget such that screen readers and other accessibility tools can adequately describe the user interface.</p>

<p>Note that elements in application screens that are purely decorative and do not provide any content or enable a user action should not have accessibility content descriptions. In this case, just suppress the lint warning with a tools:ignore=”ContentDescription” attribute.</p>

<p>Note that for text fields, you should not set both the <code>hint</code> and the <code>contentDescription</code> attributes since the hint will never be shown. Just set the <code>hint</code>. See http://developer.android.com/guide/topics/ui/accessibility/checklist.html#special-cases.</td>
    </tr>
  </tbody>
</table>